devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -394,6 +394,24 @@ bool YarpRobotLoggerDevice::open(yarp::os::Searchable& config)
         return false;
     }
 
+    // open rpc port for YarpRobotLoggerDevice commands
+    std::string portPrefix{};
+
+    if (!params->getParameter("port_prefix", portPrefix))
+    {
+
+        log()->info("{} The 'port_prefix' is not provided. It willnot be used.", logPrefix);
```

Comment:
I just noticed that, after my suggestion, if the port prefix is not specified, then the name would become too vague. Maybe it is better to specify a default value?

Reply:
probably `yarp-robot-logger` could be the right choice

Reply:
Done with [296f786](https://github.com/ami-iit/bipedal-locomotion-framework/pull/915/commits/296f78662b7ac881d78cf3c673ac94b4245e30b5)

Reply:
You can say that the default one will be used that is: and you print the port name

Reply:
Done with [db018df](https://github.com/ami-iit/bipedal-locomotion-framework/pull/915/commits/db018df80e5363a6ab336acdfffcdbad7283cbea)